docs/en/observability/inspect-panel.asciidoc

```diff
@@ -0,0 +1,75 @@
+[[inspect-panel]]
+= Inspect
+
+The *Inspect* view in {kib} allows you to view information about all requests that were made to collect the data displayed on the current page.
+
+[role="screenshot"]
+image::images/inspect-panel.png[Inspector flyover in the {uptime-app}]
+
+Many requests go into building visualizations in {kib}.
+For example, to render visualizations in the {uptime-app}, {kib} needs to request a list of all your monitors,
+data about the availability of each monitor over time, and more.
+If something goes wrong, the Inspect view can help you report an issue and troubleshoot with Elastic support.
+
+Inspecting requests is available for the following apps:
+
+* APM
+* Uptime
+* {user-experience}
+
+[discrete]
+[[inspect-panel-enable]]
+== Enable inspect
+
+To enable inspect across apps:
+
+. Go to {kib}'s {kibana-ref}/advanced-options.html[Advanced Settings].
+. Find the *Observability* section.
+. Turn on the *Inspect ES queries* option.
+. Click *Save changes*.
+
+[role="screenshot"]
+image::images/inspect-panel-enable.png[{kib} Advanced Settings {observability} section with Inspect ES queries enabled]
+
+[discrete]
+[[inspect-panel-view]]
+== Inspect view
+
+Open the inspect flyover by clicking *Inspect* in the top bar.
+
+Click the *Request* dropdown to see all the requests used to make the current page work.
+Select one to see information about the request below.
+
+[role="screenshot"]
+image::images/inspect-panel-view-dropdown.png[Inspector flyover dropdown for selecting a request to inspect]
+
+Toggle between the *Statistics*, *Request*, and *Response* tabs to see details for a single request.
+
+The *Statistics* tab provides information about the request including:
+
+[horizontal]
+Hits::                        The number of documents returned by the query.
+Hits (total)::                The number of documents that match the query.
+{data-source-cap}::           The {data-source} that connected to the {es} indices.
+{kib} API query parameters::  The query parameters used in the {kib} API request that initiated the {es} request.
+{kib} API route::             The route of the {kib} API request that initiated the {es} request.
+Query time::                  The time it took to process the query.
+                              Does not include the time to send the request or parse it in the browser.
+Request timestamp::           Time when the start of the request has been logged.
+
+[role="screenshot"]
+image::images/inspect-panel-view-statistics.png[Inspector flyover Statistics tab]
+
+The *Request* tab shows the exact syntax used in the request.
+// Example of when you might use this?
+You can click *Copy to clipboard* to copy the request or *Open in Console* to open it in the {kibana-ref}/console-kibana.html[{kib} console].
```

Comment:
Do you think it would be necessary/helpful to provide an example that illustrates the value of this feature?

Reply:
I'm not sure. Let's hear what Shahzad has to say. I know the request/response is helpful for our devs to understand and reproduce user issues (like broken APM charts), but I'm not sure what users can do with this information on their own.